src/lib/timeseries.spec.js

```diff
@@ -46,6 +50,7 @@ import {
   makeStartPlaceFeature,
   makeEndPlaceFeature
 } from 'test/mockTrip'
+const t = x => get(locales, x)
```

Comment:
(fyi: You can also do t => x => x. With that, within the test, you just get `Foo.Bar`. Like that, you can edit the translation file without the need to worry about updating the test)

Reply:
je trouvais ça sympa d'avoir les vrais trads, ça permet aussi quelque part de les tester et de gérer notamment les espaces (qu'on veut retirer/ajouter selon certains cas). Par contre en effet si la trad change, faut changer le test, pas forcément utile dans certains cas 👍